samples/python/text_generation/structured_output_generation.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import json
+from openvino_genai import LLMPipeline, GenerationConfig, StructuredOutputConfig
+from typing import Literal
+from pydantic import BaseModel, Field
+
+
+class Person(BaseModel):
+    name: str = Field(pattern=r"^[A-Z][a-z]{1,20}$")
+    surname: str = Field(pattern=r"^[A-Z][a-z]{1,20}$")
+    age: int
+    city: Literal["Dublin", "Dubai", "Munich"]
+    
+
+class Car(BaseModel):
+    model: str = Field(pattern=r"^[A-Z][a-z]{1,20} ?[A-Z][a-z]{0,20} ?.?$")
+    year: int
+    engine_type: Literal["diesel", "petrol", "electric", "hybrid"]
+
+
+class Transaction(BaseModel):
+    id: int = Field(ge=1000, le=10_000_000)
+    amount: float
+    currency: Literal["EUR", "PLN", "RUB", "AED", "CHF", "GBP", "USD"]
+
+
+class ItemQuantities(BaseModel):
+    person: int = Field(ge=0, le=100)
+    car: int = Field(ge=0, le=100)
+    transaction: int = Field(ge=0, le=100)
+
+
+items_map = {"person": Person, "car": Car, "transaction": Transaction}
+
+sys_message = (
+    "You generate jsons based on the user's request. You can generate jsons with different types of objects: person, car, transaction. "
+    "If user requested different type json fields should remain zero. "
+    "Please not that words 'individual', 'person', 'man', 'human', 'woman', 'people', 'inhabitant', 'citizen' are synonyms and can be used interchangeably. "
+    "E.g. if user wants 5 houses, then json must be {\"person\": 0, \"car\": 0, \"transactions\": 0}, "
+    "if user wand 3 person and 1 house then json must be {\"person\": 3, \"car\": 0, \"transaction\": 0}. "
```

Comment:
```suggestion
    "if user wants 3 people and 1 house then json must be {\"person\": 3, \"car\": 0, \"transaction\": 0}. "
```
(I didn't verified if the model still works correctly after that change)

Reply:
Done. Model still works. It should work even better after your grammar correction.